buildAndReleaseTask/installers/pulumi.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as tl from "azure-pipelines-task-lib/task";
+import * as lib from "azure-pipelines-tool-lib";
+import * as path from "path";
+
+/**
+ * Function installs Pulumi across Operating Systems using the Build agent's native SDK libraries. This Function does not check if the user input for versionSpec is greater than the latest available version and will result in an error if the user input was wrong.
+ * @param versionSpec version number the user wants this function to install
+ * @param latestPulumiVersion latest version based on what getLatestPulumiVersion returned
+ */
+
+export async function installPulumiWithToolLib(versionSpec: string, latestPulumiVersion: string) {
+
+  const os: string = tl.osType();
```

Comment:
Just a minor nit, but could you please check and make sure this is indented with the same number of spaces as `index.ts`? I think it is 4 spaces.

Reply:
Apologies. My default is set to 2 spaces :) I fixed this back to 4.